### PR TITLE
Update docker images version

### PR DESCRIPTION
**Pull request by DanielFran**
-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/generator-constants.js [hunk 1]
```diff
@@ -27,9 +27,9 @@ const DOCKER_MONGODB = 'mongo:4.0.12';
 const DOCKER_COUCHBASE = 'couchbase:6.0.0';
 const DOCKER_CASSANDRA = 'cassandra:3.11.4';
 const DOCKER_MSSQL = 'microsoft/mssql-server-linux:latest';
-const DOCKER_HAZELCAST_MANAGEMENT_CENTER = 'hazelcast/management-center:3.12.3';
+const DOCKER_HAZELCAST_MANAGEMENT_CENTER = 'hazelcast/management-center:3.12.4';
 const DOCKER_MEMCACHED = 'memcached:1.5.16-alpine';
-const DOCKER_KEYCLOAK = 'jboss/keycloak:6.0.1';
+const DOCKER_KEYCLOAK = 'jboss/keycloak:7.0.0';
```
Comment (vishal423) [inline generators/generator-constants.js]: we should also try to align `realm` and `user` configurations along with upgrades.

Comment (DanielFran) [reply generators/generator-constants.js]: @vishal423 In which repo?

Comment (DanielFran) [reply generators/generator-constants.js]: @vishal423 You mean here: https://github.com/jhipster/jhipster-registry/blob/master/src/main/docker/keycloak.yml

Comment (vishal423) [reply generators/generator-constants.js]: @DanielFran, we should upgrade keycloak configurations like done in this PR #9635

Comment (DanielFran) [reply generators/generator-constants.js]: @vishal423 Got it! I will revert the update here and add a note to refer the necessary changes.
Will open a new Issue for that...
Thanks

**Pull request merged**